packages/framework/client-logger/app-insights-logger/src/fluidAppInsightsLogger.ts

```diff
@@ -39,15 +39,29 @@ export interface FluidAppInsightsLoggerConfig {
 
 /**
  * Object used with an {@link FluidAppInsightsLoggerConfig}
- * to define logic for filtering of telemetry events
+ * to define a filter with logic for matching it to telemetry events.
+ * Filters can include either a category, namespace or both types of filter; A valid filter must have at least one defined.
  *
  * @public
  */
 export interface TelemetryFilter {
 	/**
 	 * The category of telemetry event that this filter applies to
 	 */
-	category: TelemetryEventCategory;
+	category?: TelemetryEventCategory;
+	/**
+	 * The namespace pattern to filter telemetry events.
+	 * For example, "perf:latency:*" will match any namespace starting with "perf:latency:"
```

Comment:
There are a few things we can do here to leverage TSDoc a bit more. Structurally, I would recommend something like:

```typescript
/**
 * The namespace pattern to filter telemetry events.
 * @remarks Expressed as a RegEx.
 * @example
 * "perf:latency:*" will match any namespace starting with "perf:latency:".
 */
```

Note that for `@example` blocks, the line containing the tag is reserved for the title of the example, so we have to put the contents on a separate line to be parsed as we expect.

Reply:
Updated.